packages/flame/lib/src/extensions/rect.dart

```diff
@@ -74,7 +74,7 @@ extension RectExtension on Rect {
     );
   }
 
-  /// Creates bounds in from of a [Rect] from a list of [Vector2]
+  /// Creates a [Rect] that represents the bounds of the list [pts].
   static Rect fromBounds(List<Vector2> pts) {
```

Comment:
This name is incorrect in my opinion. It does not create a Rect based on a given bounds it returns a Rect **that is** the bounds of the list!

So I suggest we also deprecate this and proxy it to a correctly named factory.